README.md

```diff
@@ -125,7 +124,87 @@ The following options are available:
 
 `setup`
 : A SQL statement that is executed before inserting the data, e.g. to create the target table.
-: *NOTE:* the underlying database should support the *create table if not exist* idiom, as the plugin will execute this statement every time the script is run.
+: _NOTE:_ the underlying database should support the _create table if not exist_ idiom, as the plugin will execute this statement every time the script is run.
+
+## SQL Execution Functions
+
+This plugin provides the following function for executing SQL statements that don't return data, such as DDL (Data Definition Language) and DML (Data Manipulation Language) operations.
+
+### sqlExecute
+
+The `sqlExecute` function executes a SQL statement that doesn't return a result set, such as `CREATE`, `ALTER`, `DROP`, `INSERT`, `UPDATE`, or `DELETE` statements. For DML statements (`INSERT`, `UPDATE`, `DELETE`), it returns the number of rows affected. For DDL statements (`CREATE`, `ALTER`, `DROP`), it returns `null`.
```

Comment:
> it returns the number of rows affected. 

If I'm not wrong it returns a `Map` option  

https://github.com/nextflow-io/nf-sqldb/pull/28/files#diff-4ebd7373639dfaa13beadb99204bf9b7e3b14b1250d45707e919e0e8b0a18114R188